### DISCUSSION TITLE
Default Prettier configuration

### DISCUSSION THREAD

**Discussion by carlosbaraza** [blitz-js/blitz#1390] [Ideas]
### What do you want and why?

Firstly, thank you for this great and promising project. Getting started I noticed that the new projects come with a custom Prettier configuration:

```
{
  "prettier": {
    "semi": false,
    "printWidth": 100
  }
}
```

Not to get into the semicolon war, but I think a majority of developers use the default configuration of Prettier, and it seems a bit unnecessary to enforce a custom styling.

### Possible implementation(s)

Remove the custom configuration. It might require running Prettier for the entire project on the initial bootstrap if it doesn't already.

**Comment by flybayer** [comment]
Hey @carlosbaraza thanks for the suggestion. I converted this from an issue to a discussion. Anyone else is welcome to comment. If we get overwhelming consensus that people want this removed, then happy to do that :)

**Reply by carlosbaraza** [reply]
Sounds good. Overall, I've been waiting for this project for quite some time. I really liked RoR, and I'd love a new framework leveraging modern technologies.

**Comment by clgeoio** [comment]
+1 for this idea.

**Comment by theZieger** [comment]
If it stays or goes ... I don't really care to be honest. Only thing I would like to see is something like an advanced configuration guide for Blitz inside it's docs. For let's say ... how to change prettier or load your company wide configuration, how to extend/disable eslint rules for when you really want or need to do so, etc...

**Reply by theZieger** [reply]
fyi: I created a issue for the documentation about the prettier configuration: https://github.com/blitz-js/blitz/issues/1437 If someone wants to chime in.

**Comment by nickluger** [comment]
Was also curious about the reasoning behind this.

**Comment by MrLeebo** [comment]
I agree with using the default style (semicolons added). It is marginally more future proof than relying on ASIs.
